### ISSUE TITLE
Allow attributes for relations

### ISSUE THREAD

**Issue by spyysalo** [nlplab/brat#791]
It should be possible to attach attributes to relations.

Although most of the infrastructure for doing this is already in place, some work will be needed on both server and client. I'll take this initially to look at the server side.

Tentatively marking this for 1.3.

**Comment by matiasz** [comment]
:+1: 

This feature would be very helpful for my work!

**Comment by wkiri** [comment]
I agree with matiasz - I really need this feature for my task.  Please let us know if it's still in the works! Thank you.

**Comment by gerardbdn** [comment]
I also vote for having this feature added. Thanks!

**Comment by Renaud009** [comment]
+1

**Comment by mbelousov** [comment]
+1

**Comment by janakact** [comment]
+1 

We really need this feature for our annotation task. Thank you!

**Comment by reckart** [comment]
If you need a brat-like tool that supports features on relations, you might have a look at [WebAnno](https://webanno.github.io/webanno/). Depending on your other needs, it may be a suitable alternative.

**Comment by hannahlindsley** [comment]
Does anyone actually work on this codebase...

**Comment by kanihal** [comment]
+1

**Comment by Tyriflis** [comment]
We would also benefit from having this added. WebAnno has other limitations that makes it difficult for us to use.

**Comment by wikithink** [comment]
+1
